spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -360,3 +360,212 @@ Another option to start a task using the `task-launcher` would be to create a st
 ```
 http --server.port=9000 | my-task-processor | task-launcher-local
 ```
+
+[[spring-cloud-dataflow-composed-tasks]]
+== Composed Tasks
+
+Spring Cloud Data Flow allows a user to create a directed graph where each node
+of the graph is a task application.  This is done by using the DSL for composed
+tasks.  A composed task can be created via the RESTful API, the Spring Cloud
+Data Flow Shell, or the Spring Cloud Data Flow UI.
+
+=== Creating, Launching, and Destroying a Composed Task
+==== Creating a Composed Task
+The DSL for the composed tasks is used when creating a task definition via the
+task create command. For example:
+```
+dataflow:> app register --name timestamp --type task --uri maven://org.springframework.cloud.task.app:timestamp-task:1.2.0.RELEASE
```

Comment:
It'd be nice not to have a pinned version in the registration command. Perhaps plug `<DESIRED_VERSION>` or something else to indicate it being dynamic? Also, we don't yet have `1.2.0.RELEASE`.